orderbook/src/orderbook.rs

```diff
@@ -461,6 +464,9 @@ mod tests {
         let another_sell_token = H160::from_low_u64_be(3);
         let another_balance = 200.into();
 
+        // Should not get tracked because corresponding order isn't open.
+        let untracked_sell_token = H160::from_low_u64_be(4);
```

Comment:
What is the meaning of using the token address "4" here?

Reply:
No meaning. The test just needs a couple of distinct sell token addresses and it already uses 2 and 3 a couple of lines above.